Akka.Serialization.MessagePack/Resolvers/PolymorphicFormatterResolver.cs

```diff
@@ -40,7 +40,7 @@ private IMessagePackFormatter<T> MakeValue<T>()
             if ((_normalResolver is IDoNotUsePolymorphicFormatter) == false &&
                 typeof(T).IsClass)
             {
-                if ((typeof(T).IsAbstract || typeof(T).IsSealed) &&
+                if ((typeof(T).IsAbstract || !typeof(T).IsSealed) &&
```

Comment:
Oops.

This was a nasty typo, as it actually would prevent polymorphism from working on certain property cases.